app/views/layouts/ats/_navbar.html.slim

```diff
@@ -62,18 +55,20 @@ nav.navbar.navbar-expand-lg.align-items-center.fixed-top.header.gap-2
                              type: :quick_search } \
                  )
 
-    / User icon
-    .dropdown.order-4
-      = picture_avatar_icon current_account.avatar, {},
-                            class: "d-inline-flex d-lg-none",
-                            alt: t("user_accounts.your_photo")
-      span.d-none.d-lg-block [aria-expanded="false" aria-haspopup="true"
-                              data-bs-toggle="dropdown" role="button" type="button"]
-        = picture_avatar_icon current_account.avatar, {},
-                              alt: t("user_accounts.your_photo")
-      ul.dropdown-menu.dropdown-menu-end
-        li = link_to t("core.settings"), ats_settings_path,
-                     class: "dropdown-item #{'nav-link-selected' if is_ats_profile}"
-        li
-          = button_to t("rodauth.logout_button"), rodauth.logout_path, method: :post, class: "dropdown-item",
-                      form_class: "dropdown-item", form: { data: { turbo: false } }
+    / Account icon
+    / Mobile
+    .order-4.d-lg-none
+      button.navbar-toggler [aria-controls="accountIconNav" aria-expanded="false"
+        aria-label="Toggle account icon nav" data-bs-target="#accountIconNav" data-bs-toggle="collapse" type="button"]
+        = picture_avatar_icon current_account.avatar, {}, alt: t("user_accounts.your_photo")
+    #accountIconNav.collapse.navbar-collapse.order-5
+      ul.navbar-nav
+        = account_icon_dropdown_elements
```

Comment:
I see the question in the issue, but I'm pretty sure it should work as before.
Means only avatar and no button